index.html

```diff
@@ -18,6 +18,29 @@
         <div class="line-2"></div>
         <div class="line-3"></div>
         </nav>
+        <div class="mobile-menu">
+            <div class="close"><img src="./assets/images/cancel-menu.svg" class="close-button" alt="close mobile menu"></div>
+            <div class="menu-container">
+            <p class="portfolio">
+              <a href="#works">
+                  Portfolio
+              </a>
+          </p>
+          <div class="line"></div>
+          <p class="about">
+              <a href="#about-page">
+                  About
+              </a>
+          </p>
+          <div class="line"></div>
+          <p class="contact">
+              <a href="#form-container">
+                  Contact
+              </a>
+          </p>
+          <div class="line"></div>
+        </div>
+    </div>
```

Comment:
- [ ] Good job adding the mobile menu functionality 👍🏽 . I think you are having an overflow in your mobile menu which is not making some parts of the UI clearly visible. Kindly fix that 😄 

![image](https://user-images.githubusercontent.com/63473584/188715742-d7867d11-3132-4911-930d-7ff189f72e6a.png)